package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@sonar/echoes-react",
```

Comment:
lets wait to have a final verdict on `@sonarsource` vs `@sonar`

Reply:
`@sonarsource` it is